README.md

```diff
@@ -27,16 +30,14 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 <button data-ng-click="vm.create()">Click</button>
 ```
 
-```text
-Your answer goes here...
+```In this case, there is no difference. Its a matter of preference whether we want to prefix data or directly want to use ng for angular.
 ```
```

Comment:
👍  any  HTML attribute prefixed with `data-` will be ignored by the html validator, and can therefore be used to make sure an Angular app's html can validate